src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1,4 +1,14 @@
 {
+  "fullText": {
+    "defaultDictionary": "simple"
+  },
+  "scripts": [
+    {
+      "run": "before",
+      "fromModuleVersion": "10.0.1",
```

Comment:
The next release (from mainline) of mod-circulation-storage is `10.1.0`. I think this should match that version.

If we then back port it to a bug fix release, then that might need to change.